rnaget-openapi.yaml

```diff
@@ -800,16 +1196,22 @@ components:
       properties:
```

Comment:
the `project` schema should have a `description`

Reply:
added